s3transfer/crt.py

```diff
@@ -818,13 +818,22 @@ def _default_get_make_request_args(
         on_done_before_calls,
         on_done_after_calls,
     ):
+        crt_request_type = getattr(
+            S3RequestType, request_type.upper(), S3RequestType.DEFAULT
+        )
+
+        # For DEFAULT requests, CRT requires the official S3 operation name.
+        # So transform string like "delete_object" -> "DeleteObject".
+        operation_name = None
+        if crt_request_type == S3RequestType.DEFAULT:
+            operation_name = ''.join(x.title() for x in request_type.split('_'))
+
         make_request_args = {
             'request': self._request_serializer.serialize_http_request(
                 request_type, future
             ),
-            'type': getattr(
-                S3RequestType, request_type.upper(), S3RequestType.DEFAULT
-            ),
+            'type': crt_request_type,
+            'operation_name': operation_name,
```

Comment:
How is this going to work for older releases on platforms like the P4D and TRN1 instances? Do the existing releases already allow this argument but it's not required?

Reply:
yeah, the argument has been there since last year, but it was optional and we forget to use it here.

Reply:
This was one of the last things we added last year for the boto3 integration was: https://github.com/awslabs/aws-crt-python/pull/530 Add S3ResponseError.operation_name

> Also, if user is creating a DEFAULT request (where CRT doesn't know which operation is being performed), the user can pass in the operation_name